catalog.html

```diff
@@ -15,96 +15,120 @@
-        <li>
-          <article>
-            <img src="img/amara-resort-spa.jpg" alt="Гостиница в Седоне" width="135" height="90">
-            <h3 class="place-description"><a href="#">Amara Resort & Spa</a></h3>
-            <span>Гостиница</span>
-            <span>от 4000 Р.</span>
-            <a href="#">Подробнее</a>
-            <a href="#">Забронировать</a>
-          </article>
-        </li>
-
-        <li>
-          <article>
-            <img src="img/desert-qualin.jpg" alt="Мотель в Седоне" width="135" height="90">
-            <h3 class="place-description"><a href="#">Desert Quail Inn</a></h3>
-            <span>Мотель</span>
-            <span>от 3000 Р.</span>
-            <a href="#">Подробнее</a>
-            <a href="#">Забронировать</a>
-          </article>
-        </li>
+  <div class="container">
+    <header class="header">
+      <nav class="navigation">
+        <a href="index.html">
+          <img src="img/logo.svg" alt="Логотип главной страницы" width="138" height="70">
+        </a>
+        <ul class="items main-navigation">
+          <li><a href="#">Информация</a></li>
+          <li><a href="#">Фото и видео</a></li>
+          <li><a href="#">Карта штата</a></li>
+          <li><a href="#" class="current">Гостиницы</a></li>
+        </ul>
+      </nav>
+    </header>
 
-        <li>
-          <article>
-            <img src="img/villas-at-poco-diablo.jpg" alt="Апартаменты в Седоне" width="135" height="90">
-            <h3 class="place-description"><a href="#">Villas at Poco Diablo</a></h3>
-            <span>Апартаменты</span>
-            <span>от 2000 Р.</span>
-            <a href="#">Подробнее</a>
-            <a href="#">Забронировать</a>
-          </article>
-        </li>
-      </ul>
-    </section>
-  </main>
+    <main>
+      <section>
+        <div class="container-sorts">
+          <h3 class="sort-title">Найдено:3</h3>
+          <p class="sort-item">Сортировка:</p>
+          <ul class="items sort-list">
+            <li class="sort-item"><a href="#" class="active sort-link">По цене</a></li>
+            <li class="sort-item"><a href="#" class="sort-link">По типу</a></li>
+            <li class="sort-item"><a href="#" class="sort-link">По рейтингу</a></li>
+          </ul>
+          <ul class="items switch-list">
+            <li class="switch-item">
+              <a href="#">
+                <img src="img/arrow-up.svg" alt="стрелка вверх">
+              </a>
+            </li>
+            <li class="switch-item">
+              <a href="#">
+                <img src="img/arrow-down.svg" alt="стрелка вниз">
+              </a>
+             </li>
+          </ul>
+        </div>
+        <ul class="items hotels-list">
+          <li class="hotel-item">
+            <article class="hotel-card">
+              <img src="img/amara-resort-spa.jpg" alt="Гостиница в Седоне" width="135" height="90" class="hotel-card-image">
+              <div class="hotel-card-tags">
```

Comment:
если внутри лежать разные смысловые теги, то этот див - обертка
```suggestion
              <div class="hotel-card-wrapper">
```